wdl/ONTWholeGenome.wdl

```diff
@@ -27,10 +27,10 @@ workflow ONTWholeGenome {
 
         String gcs_out_root_dir
 
-        Boolean call_variants = true
-
+        Boolean call_svs
```

Comment:
In general, I think nearly all workflow "flags" like this should be set to a reasonable default, ideally whatever settings we would use if the data was our own. Users can override it if they choose, but should they only specify input files and nothing else, I think the workflow should still run properly.

Reply:
Providing defaults making sense for our own analysis is definitely one way to go.

What's lacking, to me, is a true variant validator like [barkely](https://github.com/broadinstitute/barclay), and that should really come from the Cromwell Team